docs/src/man/intervals.md

```diff
@@ -123,3 +124,46 @@ against itself to produce "coverage intervals".
 ```@docs
 coverage
 ```
+
+
+## Random access of intervals
+
+It is often the case that the user is interested in specific regions of a file.
+In such a case, reading all data from top to the specifc regions is a
+time-wasting process when handing a big data file.
+
+[Tabix](http://www.htslib.org/doc/tabix.html) is a tool to index an interval
+data file and makes it possible to quickly retrieve data overlapping with a
+specific genomic region. Indexing a file using the tabix tool requires that the
+data file is sorted and then compressed in a special file format called
+[BGZF](https://github.com/BioJulia/BGZFStreams.jl). The BGZF file format is
+gzip-compliant data format and readable using standard command-line tools like
```

Comment:
Thank you! I often forget to insert an article since the Japanese language doesn't have a counterpart of it.